### PR TITLE
Adds 5th batch of 4.8 RN bug doc text

### PR DESCRIPTION
**Pull request by stevsmit**
For 4.8.

No BZ

Preview: https://deploy-preview-34289--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -1577,6 +1579,21 @@ This release removes the Prometheus Adapter, which was a Technology Preview feat
...
+
+* Previously, nodes appeared unhealthy and did not operate as expected during periods of high workloads. This resulted from workloads using memory faster than the memory could be reclaimed. With this update, memory reclamation and out-of-memory situations were addressed and these conditions no longer occur during high workload situations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1931467[*BZ#1931467*])
+
+* Previously, the maximum transmission unit (MTU) specification for a bond interface using kernal arguments did not get assigned properly. This has been corrected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1932502[*BZ#1932502*])
+
+* Previously, the `clevis-luks-askpass.path` unit was not enabled by default. This caused non-root `LUKS Clevis` devices to fail to unlock automatically on reboot. TThis update enables the `clevis-luks-askpass.path` unit by default and allows non-root `LUKS Clevis` devices to unlock automatically on reboot. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1947490[*BZ#1947490*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-8-release-notes.adoc:1594]: s/TThis/This

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -1614,6 +1631,20 @@ uid=1001(1001) gid=1001 groups=1001
 
 * Previously, the Fibre Channel volume was incorrectly unmounted from a node when a pod was deleted. This happened when a different pod that used the volume was deleted in the API server when the kubelet on the node was not running. With this update, Fibre Channel volumes correctly unmount when a new kubelet starts. Additionally, the volume cannot be mounted to multiple nodes until the new kubelet fully starts and cofnrims that the volume is unmounted, which ensures that Fibre Channel volumes are uncorrupted. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954509[*BZ#1954509*])
 
+*Windows Containers*
+
+* Previously, the load balancer service became unstable when users would scale additional Windows nodes. With this update, the load balancer service is stabilized, which allows users to add multiple Windows nodes without erratic performance.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1905950[*BZ#1905950*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-8-release-notes.adoc:1636]: Looks like a missing space after the period here

performance. (link:

File: release_notes/ocp-4-8-release-notes.adoc [hunk 3]
```diff
@@ -1614,6 +1631,20 @@ uid=1001(1001) gid=1001 groups=1001
...
+
+* Previously, when users provided an invalid private key, the Windows Machine Config Operator (WMCO) would fail. With this update, the WMCO produces an error alerting the user of an invalid key. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929579[*BZ#1929579*])
+
+* Previously, the kube-proxy service crashed unexpectedly after the load balancer was created if it was created after the Windows pods were running. With this update, the kube-proxy service does not crash when recreating the load balancer service. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939968[*BZ#1939968*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-8-release-notes.adoc:1640]: `kube-proxy` service

**Comment by ahardin-rh** [comment]
Just a few minor comments :smile:

**Review by bobfuru — APPROVED** [review]

**Pull request merged**